Makefile

```diff
@@ -58,8 +41,9 @@ generate: ## autogenerate go files for config
 	go generate ./config/configuration.go
 
 install-subkey: ## installs subkey
-	curl https://getsubstrate.io -sSf | bash -s -- --fast
-	cargo install --force --git https://github.com/paritytech/substrate subkey
+	wget -P ${HOME}/.cargo/bin/ https://storage.googleapis.com/centrifuge-dev-public/subkey-rc6
```

Comment:
Keep in mind that this will only work for a linux distribution, prob not for macOS, in which case cargo install would still help to keep in, although we are going to start using go-subkey-port so, might not be needed at all.

Reply:
This is TODO yet. i'm not able to upgrade the gsrpc because for latest go-ethereum issue. Once that is fixed, mac os should run natively. Until then, we anyway throw a helpful error so its easy to identify the cause